narwhals/_duckdb/namespace.py

```diff
@@ -157,6 +158,16 @@ def func(df: DuckDBLazyFrame) -> list[duckdb.Expression]:
             kwargs={"exprs": exprs},
         )
 
+    def when(
+        self,
+        *predicates: IntoDuckDBExpr,
```

Comment:
```suggestion
        *predicates: IntoDuckDBExpr
```

trailing comma 😱 (just joking. I love the other PRs 😅)

Reply:
ok this made me laugh out loud 😆